tensorboard/webapp/BUILD

```diff
@@ -342,7 +342,6 @@ tf_ng_web_test_suite(
         "//tensorboard/webapp/runs/store:testing",
         "//tensorboard/webapp/runs/views/runs_selector:runs_selector_test",
         "//tensorboard/webapp/runs/views/runs_table:runs_table_test",
-        "//tensorboard/webapp/settings:test_lib",
```

Comment:
There might've one a long time ago that I don't remember, but was there an offline discussion about moving away from the hourglass-style for testing targets?

Reply:
Hourglass makes any changes in webapp folder painfully slow for `//tensorboard/webapp:karma_test` so we decided to have per-feature "hourglass".